test/units/parsing/test_ajson.py

```diff
@@ -20,3 +24,66 @@ def test_AnsibleJSONDecoder_vault():
     assert isinstance(data['password'], AnsibleVaultEncryptedUnicode)
     assert isinstance(data['bar']['baz'][0]['password'], AnsibleVaultEncryptedUnicode)
     assert isinstance(data['foo']['password'], AnsibleVaultEncryptedUnicode)
+
+
+class TestAnsibleJSONEncoder():
```

Comment:
@mattclay , yes. I’d like to organize tests for each class of ajson.py by particular classes into the test file.
If new classes were added to ajson.py, I think, it will be more comfortable to work with test in this way

Reply:
@Andersson007 it's [a premature optimization](https://stackify.com/premature-optimization-evil/). Currently, the class is unnecessary.